feature/bgp/otg_tests/bgp_override_as_path_split_horizon_test/bgp_override_as_path_split_horizon_test.go

```diff
@@ -321,11 +325,9 @@ func testSplitHorizonAllowOwnAs3(t *testing.T, args *otgTestArgs) {
 
 	t.Run("Re-advertise the prefix from the ATE with 1 Occurrence: 65500 dutLocalAS1 65499", func(t *testing.T) {
 		advBGPRouteFromOTG(t, args, []uint32{65500, dutLocalAS1, 65499})
-
```

Comment:
keep these blank lines;

These blank lines are generally given to separate the context and improve the method readability.

Reply:
Addressed the comment in the new commit